CoordgenMacrocycleBuilder.h

```diff
@@ -151,16 +152,16 @@ class
     Polyomino
 {
   public:
-    Polyomino();
+    EXPORT_COORDGEN Polyomino();
     Polyomino(const Polyomino& p);
-    ~Polyomino();
+    EXPORT_COORDGEN ~Polyomino();
     Polyomino& operator=(const Polyomino& rhv);
 
     /*
      explore the topology of the polyominoes and returns true if they have the same. Takes
      into account translations, rotations and mirroring
      */
-    bool isTheSameAs(Polyomino& p) const;
+    bool EXPORT_COORDGEN isTheSameAs(Polyomino& p) const;
 
     /*returns the number of hexagons in the polyomino*/
     int size() const;
```

Comment:
What about this? Is there a reason not to include this in the public API?

Basically, if part of the class is public, I don't see why the whole class shouldn't be public. What do you gain by keeping the rest of this private?

Reply:
I set the flag on each class that had at least a method marked as EXPORT_COORDGEN

Reply:
I think that makes sense. It probably makes sense to export the whole public API. Any methods that we don't want exported could be private.